operator/pkg/bgpv2/manager.go

```diff
@@ -247,13 +239,30 @@ func (b *BGPResourceManager) Run(ctx context.Context) (err error) {
 
 // reconcileWithRetry retries reconcile with exponential backoff.
 func (b *BGPResourceManager) reconcileWithRetry(ctx context.Context) error {
+	// steps will repeat for ~8.5 minutes.
```

Comment:
@harsimran-pabla I also moved this backoff config into the reconcile func as that seems to be the pattern in the cilium codebase, and makes it easier to relate with the retryFn (how many attempts we are actually doing).